packages/lib/server/repository/webhook.ts

```diff
@@ -81,15 +81,15 @@ export class WebhookRepository {
     let userWebhooks = user.webhooks;
     userWebhooks = userWebhooks.filter(filterWebhooks);
     let webhookGroups: WebhookGroup[] = [];
-    const bookerUrl = await getBookerBaseUrl(organizationId ?? null);
```

Comment:
@retrogtx It was used to construct `bookerUrl` which is no longer required, so removed it